docs/articles/2023/20230908-out_of_core_methods.md

```diff
@@ -0,0 +1,133 @@
+# Memory-efficient implementations of commonly used single-cell methods
+
+*Published:* *September [DAY], 2023*
+
+*By:* *[Pablo Garcia-Nieto](pgarcia-nieto@chanzuckerberg.com)*
+
+The Census team is thrilled to officially announce the first roll out of memory-efficient implementations of some of the most widely used single-cell algorithms.
+
+With just a few lines of code, using the Census Python API, users can now perform the following processing tasks in a regular laptop with 8GB of memory across dozens of millions of cells:
+
+* Calculating average and variance gene expression for cells or genes. See this [tutorial](../../notebooks/experimental/mean_variance.ipynb).
+* Obtaining batch-corrected highly variable genes. See this [tutorial](../../notebooks/experimental/highly_variable_genes).
```

Comment:
Maybe something like "See example below or the full tutorial"

Reply:
I would also add "for general information on querying the Census, see our introductory documentation at..."

I.e., assume users may arrive here via a google search, and need a back-pointer